docs/src/components/ui/Banner.tsx

```diff
@@ -123,27 +130,24 @@ const BracesAnimation: React.FC<React.PropsWithChildren<{}>> = ({
   );
 };
 
-const DefaultBanner = ({ content }: DefaultBannerProps) => {
-  return (
-    <BracesAnimation>
-      {content.text.split(" ").map((word: string, index: number) => (
```

Comment:
> 1 When it's possible try to keep logic in the function body while the returning JSX expression should be as simple as possible

2 Do we have a reason to wrap each word into a span?

3 TODO: we can later switch to markdown to format text